modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,258 @@
+
+After you have created users, they can upload files and share these with other ownCloud users and guest users. Folders can be created and shared as well. Sharing works via the email address of the user who should be granted access to certain files.
+
+Once a regular user shares a file or folder specifying an external email address, the recipient of the email becomes a guest user.
+
+== Enabling ownCloud Apps
+
+With the free community edition, you can use all apps that are published under the GPL or similar public licenses, e.g. the GNU Affero General Public License. These apps you can simply enable and enjoy.
+In the upper left corner, click on the main menu with the three bars, select menu:Market and install what you like.
+
+image:qnap/ownCloud_main_menu.png[ownCloud Main Menu]
+
+A bundle of Enterprise Apps is published under the ownCloud Commercial License and only available with the Enterprise Edition. To see what's not included in the free community edition, select `App Bundles` in the left-hand navigation bar.
+
+== Enterprise Edition and Licensing
+
+If you want to use enterprise features, obtain a license from the QNAP Store and activate it in the QNAP LicenseCenter. In case you want to have more regular users, you can buy additional licenses.
+
+Licenses are valid for a year. If you don't renew them, only the first five users created will remain enabled and only non-enterprise apps. Should you decide to buy licenses again at a later point in time, you can enable the users you want to become active again in the web interface.
+
+Users can also be enabled or disabled via `occ` commands. For more information on the ownCloud command line interface, see below. This would be particularly useful if the admin user accidentally gets disabled.
+
+== Accessing your QNAP ownCloud from the Internet
+
+If you want to connect to your ownCloud on QNAP from the Internet, you need to configure the network accordingly. In the menu:Main menu of your QNAP NAS, select menu:SYSTEMS[Network & Virtual Switch]. Under "Access Services" click on menu:DDNS (Dynamic Domain Name Service) then btn[Add]. Here you can configure the DDNS settings.
+
+image:qnap/DDNS_qnap-cropped.png[DDNS Configuration on QNAP]
+
+In case you want to assign a static IP address, you need to access your QNAP device via `ssh` on the command line and edit the file `custom/user.config.php` in your top-level ownCloud app directory, e.g. `/share/CACHEDEV1_DATA/.qpkg/ownCloud`.
+Create an entry like in the following example with the correct IP address:
+
+[source,console]
+----
+<?php
+$CONFIG = array(
+    'overwriteprotocol' => 'https',
+    'overwritehost' => '203.0.113.0',
+    'overwrite.cli.url' => 'https://203.0.113.0/owncloud/',
+);
+----
+
+If you're using a subdomain instead of a subdirectory for ownCloud, adjust the URL accordingly with the correct prefix, e.g. `https://owncloud/203.0.113.0/`.
```

Comment:
```suggestion
If you're using a subdomain instead of a subdirectory for ownCloud, adjust the URL accordingly with the correct prefix, e.g. `https://203.0.113.0/owncloud/`.
```

The example looks the wrong way around to me.

Reply:
No, this was referring to subdomains could.yourdomain.com setup, but it should probably go.